tests/upgrades/test_satellitesync.py

```diff
@@ -0,0 +1,136 @@
+"""Test for Inter Satellite Sync related Upgrade Scenario's
+
+:Requirement: Upgraded Satellite
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: API
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from fauxfactory import gen_string
+from nailgun import entities
+from robottelo import ssh
+from robottelo.cli.contentview import ContentView
+from robottelo.cli.package import Package
+from robottelo.decorators import pre_upgrade, post_upgrade
+from robottelo.test import CLITestCase
+from upgrade_tests.helpers.scenarios import create_dict, get_entity_data
+
+
+class scenario_preversion_cv_exports_imports(CLITestCase):
+    """Test Content-view created before upgrade can be exported and imported after upgrade
+
+        :id:
+
+        :steps:
+
+        1. Before upgrade, Create a ContentView
+        2. Publish and promote the CV
+        3. Upgrade the satellite
+        4. Post upgrade, Import and Export the Content Views version created before upgrade
+
+        :expectedresults:
+
+        1. Content-view created before upgrade should be exported and imported after upgrade
+     """
+    export_base = '/var/lib/pulp/katello-export/'
+
+    def tearDownScenario(self):
+        """Removes CV export file/directory from export base directory"""
+        ssh.command('rm -rf {}/*'.format(self.export_base))
+
+    def set_importing_org(self, product, repo, cv, mos='no'):
```

Comment:
```suggestion
    def set_importing_org(self, product, repo, cv):
```
As it's mandatory to have this setting off hence can we directly pass 'no' under below ..

Reply:
It is required for negative tests, but I can remove it for now.